src/gfn/gflownet/detailed_balance.py

```diff
@@ -66,19 +72,20 @@ def get_scores(self, env: Env, transitions: Transitions) -> Tuple[
 
         if states.batch_shape != tuple(actions.batch_shape):
             raise ValueError("Something wrong happening with log_pf evaluations")
-        if not self.off_policy:
+        if (
+            transitions.log_probs is not None
```

Comment:
I'm seeing this logic a few times in the code. Should we abstract it into a utility like 
```
def has_log_probs(obj):
    return obj.log_probs is not None and obj.log_probs.nelement() > 0
```
?

Reply:
I've added this utility function.